common/src/jwt_managers.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime, timedelta, timezone
+
+import jwt
+
+import common.src.settings as settings
+
+
+class JWTManager:
+    def __init__(self,
+                 private_key: str | None,
+                 public_key: str | None,
+                 algorithm: str,
+                 expiration_time_minutes: int | None):
+        self.private_key = private_key
+        self.public_key = public_key
+        self.algorithm = algorithm
+        self.expiration_time_minutes = expiration_time_minutes
+
+    def generate_token(self, payload_arg: dict) -> (bool, str | None, list[str] | None):
```

Comment:
In my opinion, this function should be renamed `generate_jwt()` to keep the same syntax.

Reply:
https://github.com/tdameros/42-transcendence/pull/96/commits/deb603477bb8f058ae2f03f3690e7c6ab06f18a5